--- links-hacked/font_builtin.c.orig	2003-05-05 02:41:09.000000000 +0200
+++ links-hacked/font_builtin.c	2004-10-08 17:28:46.000000000 +0200
@@ -105,9 +105,10 @@
 	 */
 	number_of_passes=png_set_interlace_handling(png_ptr);
 	png_read_update_info(png_ptr,info_ptr);
-	*dest=mem_alloc(*x*(*y));
+	/* needs buffer for pixel depth before transform */
+	*dest=mem_alloc(*x*(*y)*png_ptr->pixel_depth/8);
 	ptrs=mem_alloc(*y*sizeof(*ptrs));
-	for (y1=0;y1<*y;y1++) ptrs[y1]=*dest+*x*y1;
+	for (y1=0;y1<*y;y1++) ptrs[y1]=*dest+(*x*png_ptr->pixel_depth/8)*y1;
 	for (;number_of_passes;number_of_passes--){
 		png_read_rows(png_ptr, ptrs, NULL, *y);
 	}
